PushNotifications/PushNotifications/Instance.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+
+struct Instance {
+    static func persist(_ instanceId: String) throws {
+        guard let savedInstanceId = Instance.getInstanceId() else {
+            UserDefaults(suiteName: "PushNotifications")?.set(instanceId, forKey: "com.pusher.sdk.instanceId")
+            return
+        }
+
+        if(instanceId != savedInstanceId) {
```

Comment:
Nitpick: Don't need the parens

Reply:
Also, given you `throw` in here I'd suggest it might make more sense being  `guard instanceId == savedInstanceId else {`